images/kube-api-rewriter/local/test-controller/go.mod

```
@@ -1,4 +1,4 @@
-module kube-api-proxy-test-controller
+module kube-api-rewriter-test-controller
 
 go 1.21
```

Maybe we should use version 1.22.7 here?
```suggestion
go 1.22.7
```